treeherder/webapp/api/intermittents_view.py

```diff
@@ -94,13 +88,10 @@ def get_queryset(self):
                                          'date', 'test_runs')
 
         if bug_id:
-            job_query = BugJobMap.objects.filter(job__repository_id__in=repo,
-                                                 job__push__time__range=(startday, endday),
-                                                 job__failure_classification__id=4,
-                                                 bug_id=int(bug_id)
-                                                 ).select_related('push').annotate(date=TruncDate('job__push__time'))\
-                                                 .values('date').annotate(failure_count=Count('id')).order_by(
-                                                 'date').values('date', 'failure_count')
+            job_query = BugJobMap.failures.default(repo, startday, endday).by_bug(bug_id).annotate(
```

Comment:
If you wrap the QuerySet in parentheses then you can lay it out like so:

```
job_query = (BugJobMap.failures.default(repo, startday, endday)
                               .by_bug(bug_id)
                               .annotate(date=TruncDate('job__push__time'))
                               .values('date')
                               .annotate(failure_count=Count('id'))
                               .order_by('date')
                               .values('date', 'failure_count'))
```

This put each method in the chain in a list so the reader can work their way top to bottom.

Reply:
Thanks for the tip!